### PR TITLE
Unfinity 7 cards

### PR DESCRIPTION
**Pull request by Simisays**
Probably the last easy Unfinity cards left. If some dev would find the time to implement the mechanics, I woudn't mind scripting more cards. 

Still in draft because Resolute Veggiesaur condition isn't working (is it even possible without engine changes?) and Pair o' Dice Lost changezone needs some more work. Though to just put it out there for some advice

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Resolute Veggiesaur
+ManaCost:2 G
+Types:Creature Dinosaur Employee
+PT:3/3
+K:Trample
+T:Mode$ RolledDie | ValidPlayer$ You | TriggerZones$ Battlefield | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ3 | Execute$ TrigPutCounter | TriggerDescription$ Whenever you roll your third die each turn, put a +1/+1 counter on CARDNAME.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt:6]: Conditions aren't for triggers

File: forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Slight Malfunction
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Charm | Choices$ DBDestroyArtifact,DBRoll
+SVar:DBDestroyArtifact:DB$ Destroy | ValidTgts$ Artifact | SpellDescription$ Destroy target artifact.
+SVar:DBRoll:DB$ RollDice | ResultSVar$ X | SubAbility$ DBDamage | SpellDescription$ Roll a six-sided die. When you do, CARDNAME deals 1 damage to each of up to X target creatures, where X is the result.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt:6]: reflexive trigger

File: forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt:6]: Tried it like that but didn't work. Probably need engine changes

File: forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt:6]: Why resolve, it's currently wrong?

File: forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/resolute_veggiesaur.txt:6]: No, doesn't need any

File: forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt:6]: Woops seem to have forgot to push my commit.

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/one_clown_band.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:One-Clown Band
+ManaCost:1 R
+Types:Artifact Creature Clown Robot Bard
+PT:1/3
+A:AB$ Pump | Cost$ 2 R | ValidTgts$ Robot | NumAtt$ +2 | TgtPrompt$ Select target Robot | KW$ Lifelink | SpellDescription$ Target Robot gets +2/+0 until end of turn.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/one_clown_band.txt:5]: why the lifelink?

File: forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Pair o' Dice Lost
+ManaCost:3 G G
+Types:Instant
+A:SP$ RollDice | Amount$ 2 | ResultSVar$ Y | SubAbility$ DBChange | SpellDescription$ Roll two six-sided dice. Return any number of cards with total mana value X or less from your graveyard to your hand, where X is the total of those results. Exile CARDNAME.
+SVar:DBChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ChangeNum$ X | WithTotalCMC$ Y | SelectPrompt$ choose any number of cards with total mana value X or less from your graveyard to your hand, where X is the total of those results | Hidden$ True | AnyNumber$ True | ChangeType$ Card.YouOwn | SubAbility$ DBExile
+SVar:DBExile:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | StackDescription$ Exile CARDNAME
+SVar:X:Count$ValidGraveyard Card.YouOwn
+DeckHints:Ability$Graveyard|Mill
+DeckHas:ABility$Graveyard
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt:9]: `ABility` > `Ability`

File: forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Pair o' Dice Lost
+ManaCost:3 G G
+Types:Instant
+A:SP$ RollDice | Amount$ 2 | ResultSVar$ Y | SubAbility$ DBChange | SpellDescription$ Roll two six-sided dice. Return any number of cards with total mana value X or less from your graveyard to your hand, where X is the total of those results. Exile CARDNAME.
+SVar:DBChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ChangeNum$ X | WithTotalCMC$ Y | SelectPrompt$ choose any number of cards with total mana value X or less from your graveyard to your hand, where X is the total of those results | Hidden$ True | AnyNumber$ True | ChangeType$ Card.YouOwn | SubAbility$ DBExile
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt:5]: `choose` > `Choose`

File: forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Name:Slight Malfunction
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Charm | Choices$ DBDestroyArtifact,DBRoll
+SVar:DBDestroyArtifact:DB$ Destroy | ValidTgts$ Artifact | SpellDescription$ Destroy target artifact.
+SVar:DBRoll:DB$ RollDice | ResultSVar$ X | SubAbility$ DBImmediateTrig | SpellDescription$ Roll a six-sided die. When you do, CARDNAME deals 1 damage to each of up to X target creatures, where X is the result.
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | ConditionCheckSVar$ X | Execute$ TrigDamage
+SVar:TrigDamage:DB$ DealDamage | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature | NumDmg$ 1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt:8]: TgtPrompt$ Select up to X target creatures

File: forge-gui/res/cardsfolder/upcoming/priority_boarding.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Name:Priority Boarding
+ManaCost:2 R
+Types:Enchantment
+T:Mode$ RolledDie | TriggerZones$ Battlefield | Execute$ TrigReveal | ValidPlayer$ You | OptionalDecider$ You | ActivationLimit$ 1 | TriggerDescription$ Whenever you roll a die, you may reveal the top card of your library. Do this only once each turn. Whenever you reveal a card with mana value less than the result this way, you may exile it. If you do, you may play it this turn.
+SVar:TrigReveal:DB$ PeekAndReveal | NoPeek$ True | RememberRevealed$ True | SubAbility$ DBExile
+SVar:DBExile:DB$ ChangeZone | Optional$ True | Defined$ Remembered | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBEffect
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/priority_boarding.txt:6]: the mana value check should affect being able to exile the card as well

File: forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt (hunk 5 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/pair_o_dice_lost.txt:5]: Hmm, once this is in I might make the SelectPrompt parse Y

**Review by tool4ever — APPROVED** [review]

**Pull request merged**